src/components/UI/HeaderHeart/HeaderHeart.tsx

```diff
@@ -13,14 +13,24 @@ export default function HeaderHeart() {
     <Link href={'/favourites'}>
       <button
         className={
-          'relative flex items-center gap-2 rounded-full px-4 py-2 font-medium text-primary'
+          'relative flex items-center  gap-2 rounded-full px-4 font-medium text-primary'
         }
       >
```

Comment:
Maybe we shouldn't wrap the 'button' tag in an 'a' tag. I think its not good for accessibility because when the user presses tab he will go to the same place twice and According to HTML 5 it is not recommended to use nested interactive elements. I think we should just use Link and remove 'button'. If you agree you can make changes here and in another header elements.

Reply:
fixed